regions/core/core.py

```diff
@@ -291,3 +291,49 @@ def to_pixel(self, wcs):
             The world coordinate system transformation to assume
         """
         raise NotImplementedError
+
+
+class RegionMeta(dict):
+    """
+    A python dictionary subclass which holds the meta attributes of the region.
+    """
+    valid_keys = ['label', 'symbol', 'include', 'frame', 'range', 'veltype',
```

Comment:
Could you add a note above here telling the origin of these keys?  I think they come from one of the CASA CRTF specification documents?

Reply:
ya, sure. yes, these are from the specification docs.